aten/src/ATen/native/cuda/SpectralOps.cpp

```diff
@@ -304,6 +304,14 @@ static const Tensor& _exec_fft(Tensor& out, const Tensor& self, IntArrayRef out_
   CUFFT_CHECK(cufftSetWorkArea(plan, workspace.mutable_data_ptr()));
 
   // execute transform plan
+  CUcontext pctx = nullptr;
+  at::globalContext().getNVRTC().cuCtxGetCurrent(&pctx);
+  if (!pctx) {
+    // workaround for corner case where a primary context exists but is not
+    // the current context
+    at::globalContext().getNVRTC().cuDevicePrimaryCtxRetain(&pctx, 0);
```

Comment:
Do you mind printing a warning here (as correct me if I'm wrong almost not real app should ever get there, should they?)